cscs-checks/tools/io/nco.py

```diff
@@ -130,9 +128,7 @@ def __init__(self):
         ]
         self.sanity_patterns = sn.all([
             sn.assert_not_found(r'(?i)unsupported|error', self.stderr),
-            sn.assert_found(r':physics.*"Modified ECMWF physics"|'
-                            r'^Global attribute \d+: CDO, size = 63 NC_CHAR',
-                            self.stdout)
+            sn.assert_found(r'physics.*Modified ECMWF physics.*', self.stdout)
```

Comment:
The second `.*` should not be necessary:
`sn.assert_found(r'physics.*Modified ECMWF physics', self.stdout)`